capa/main.py

```diff
@@ -670,9 +672,14 @@ def main(argv=None):
         with open(args.sample, "rb") as f:
             extractor = capa.features.freeze.load(f.read())
     else:
-        format = args.format
+        if args.format == "auto" and args.sample.endswith(EXTENSIONS_SHELLCODE_32):
```

Comment:
what is this magic? you can pass a sequence to `endswith` to check for any of the values???

Reply:
![image](https://user-images.githubusercontent.com/156560/114076811-5418e000-9864-11eb-9a6e-f6c99dbf8218.png)
my mind is blown!